README.md

```diff
@@ -14,14 +14,15 @@
 **Do note that in certain operating systems Heroku CLI doesn't get added to path automatically. If that's the case you'll have to add heroku to your path manually.**
 - Run this command in cmd or powershell `heroku stack:set container -a appname`, where `appname` is replaced with your Heroku app's name.
 - In heroku, in the `Deploy` tab click on `Connect to GitHub` and then click on `search` and select your fork/import of this repository.
-- Now scroll down and under `Manual deploy`, click on `deploy` with the `master` branch selected.
+- Now scroll down and under `Manual deploy`, click on `deploy` with the `master` branch selected. <br/>
+**Do note that you could also `Enable Automatic Deploys` with the `master` branch selected if you would like each commit you make to get automatically and easily deployed onto your bot. It is your choice whether you'd like to Enable or Disable Automatic Deploys.**
```

Comment:
I’ve mentioned it’s optional @OIVAS7572

Reply:
if it is optional no need to wright it as bold
I suggest to put it in a brackets